src/controller/controller.cpp

```diff
@@ -265,31 +311,21 @@ void NWNXController::notifyServiceShutdown()
 
 bool NWNXController::startServerProcessInternal()
 {
-    SHARED_MEMORY shmem;
-	std::string nwnexe("\\nwn2server.exe");
-	auto pszHookDLLPath = "NWNX4_Hook.dll";
-
 	ZeroMemory(&si, sizeof(si));
 	ZeroMemory(&pi, sizeof(pi));
 	si.cb = sizeof(si);
 
-	auto exePath = nwninstalldir + nwnexe;
-	logger->Trace("Starting server executable %s in %s", exePath.c_str(), nwninstalldir.c_str());
+	auto exePath = m_nwn2InstallDir / "nwn2server.exe";
```

Comment:
That is cool! I didn't realize they overrode the / operator for paths like that.

Reply:
It's a bit disturbing the first time you see it, but it's really practical :)